sdk/tests/js/tests/tex-image-and-sub-image-2d-with-image.js

```diff
@@ -43,7 +43,7 @@ function generateTest(pixelFormat, pixelType, pathToTestRoot, prologue) {
         gl.clearColor(0,0,0,1);
         gl.clearDepth(1);
 
-        wtu.loadTexture(gl, "/resources/red-green.png", runTest);
+        wtu.loadTexture(gl, "../../resources/red-green.png", runTest);
```

Comment:
It's a good suggestion. Let's see how difficult this new structure is to maintain. If we can avoid adding too much intelligence to the resource finding code, that's probably better in the long run.